tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+# Sharing fixtures
+# Ref: https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session
+
+import os
+import subprocess
+import shlex
+import tempfile
+import pytest
+import git
+
+git_extras_cwd = os.getcwd()
+
+@pytest.fixture(scope="module")
+def git_repo():
+    tmp_dir = tempfile.mkdtemp()
+    tmp_file_a = tempfile.mkstemp(dir=tmp_dir)
+    tmp_file_b = tempfile.mkstemp(dir=tmp_dir)
+    repo = git.Repo.init(tmp_dir)
+    os.chdir(tmp_dir)
+    repo.git.add(".")
+    repo.git.config("--local", "user.name", "test")
+    repo.git.config("--local", "user.email", "test@git-extras.com")
+    repo.git.commit("-m", "chore: initial commit")
+    return [repo, tmp_dir, tmp_file_a, tmp_file_b]
```

Comment:
Would be better to return an object, which provides some method like `.create_tmp_file`. So people don't need to remember the argument order.

Reply:
Got the idea.